### ISSUE TITLE
'ClientConnection' object has no attribute 'get_sessionmsg'

### ISSUE THREAD

**Issue by david-shmailov** [breakerspace/bento#8]
Im trying to recreate your paper evaluations,  but every script that uses get_sessionmsg function crashes due to it not being defined anywhere. i did a thorough search and although the docs/api.html defines "get_sessionmsg" as part of the api, it does not appear anywhere in the client/api.py file. ive tried to look in other libraries and found nothing. what am i missing here?
Thanks in advance
![image](https://user-images.githubusercontent.com/13394949/150643746-5d5cdbf6-f085-4c6b-9dcd-e6dba4a4d386.png)

**Comment by Thousanrance** [comment]
You can try downloading the code under the branch 'sigcomm21' instead of 'main'.

**Comment by DrNeenu** [comment]
Yes, but the issue still persists. Any help will be appreciated.
